db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
+EVAL_LOCAL = "local"
+EVAL_REMOTE = "remote"
 # Filter types are defined in `eval_filter_type` type. See schema definition.
 FILTER_ARTIST = "artist"
 
 
-def evaluate_dataset(dataset_id, normalize, filter_type=None):
+def evaluate_dataset(dataset_id, normalize, filter_type=None, eval_location=EVAL_LOCAL):
```

Comment:
As a side note, in Python 3 there is a feature that lets you force to explicitly specify keyword arguments. https://www.python.org/dev/peps/pep-3102/